website/docs/reference/resource-configs/severity.md

```diff
@@ -29,18 +29,18 @@ Here's how those play in practice:
 Note that test warn statuses will return errors instead if the [`--warn-error`](global-cli-flags#warnings-as-errors) flag is passed. Unless dbt is told to treat warnings as errors, a test with `warn` severity will never return an error.
 
 <Tabs
-  defaultValue="specific"
+  defaultValue="generic"
   values={[
-    { label: 'Specific test', value: 'specific', },
-    { label: 'One-off test', value: 'one_off', },
-    { label: 'Generic test block', value: 'generic', },
+    { label: 'Built in generic tests', value: 'generic', },
```

Comment:
We use "[out of the box" if you scroll down here](https://deploy-preview-1011--docs-getdbt-com.netlify.app/docs/building-a-dbt-project/tests#generic-tests), so I wonder if we should stick with that and say "out-of-the-box generic tests," which is longer 🤔